meeting-notes/2018/2018-09-10--js-core-dev-team-weekly.md

```diff
@@ -0,0 +1,144 @@
+# ⚡️ⒿⓈ Core Dev Team Weekly Sync 🙌🏽 September 10, 2018
+
+- **Lead:** Alan Shaw
+- **Notetaker:** @jacobheun
+- **Attendees:**
+  - @alanshaw
+  - @achingbrain
+  - @travisperson
+  - @jacobheun
+  - @vasco-santos
+  - @vmx
+- **Recording:** https://youtu.be/KhsdFb_b8Yo
+
+## Agenda
+
+- Ask everyone to put their name into the list of attendees
+- Round of updates
+  - What have you accomplished since the last Weekly?
+  - Were there any blockers? If so, which ones? Is it still blocked? Why?
+  - What is the next important thing you should focus on?
+- Ask for general questions. Could be things like:
+  - I'm stuck with something, I don't know who to ask. Who knows who to ask?
+  - Who can help me with xyz?
+- Plan this week
+  - Select issues to work on
+- Review remaining issues if there is time left
+
+## Notes
+
+### Week Update
+
+@alanshaw
+- Done:
+    - Published release candidate for `js-ipfs` 0.32.0-rc.1 and 0.32.0-rc.2
+    - Test stabilisation work [#1541](https://github.com/ipfs/js-ipfs/pull/1541)
```

Comment:
Nice! We should avoid the global flags though, they will hide tests that are slow when they shouldn't be.